### PR TITLE
[CoreBundle] Fixing ResourceQueryBuilder initialization.

### LINKED ISSUE #630: bug : multiple Teams
i've created a team ("create teams" => i choose to create 1 team, name : team => creates "team 1")
i've create  another team ("create teams" => i choose to create 2 teams, name : teamX => i have a error)
```
Catchable Fatal Error: Argument 1 passed to Claroline\CoreBundle\Manager\ResourceManager::getLastIndex() must be an instance of Claroline\CoreBundle\Entity\Resource\ResourceNode, null given, called in /home/user/www/claroline6/claro6-path-lock-dev/Claroline/vendor/claroline/distribution/plugin/team/Manager/TeamManager.php on line 565 and defined
500 Internal Server Error - ContextErrorException

Stack Trace
in vendor/claroline/distribution/main/core/Manager/ResourceManager.php at line 1813   -
        $this->om->endFlushSuite();
    }
    public function getLastIndex(ResourceNode $parent)
    {
        try {
            $lastIndex = $this->resourceNodeRepo->findLastIndex($parent);
at ErrorHandler ->handleError ('4096', 'Argument 1 passed to Claroline\CoreBundle\Manager\ResourceManager::getLastIndex() must be an instance of Claroline\CoreBundle\Entity\Resource\ResourceNode, null given, called in /home/user/www/claroline6/claro6-path-lock-dev/Claroline/vendor/claroline/distribution/plugin/team/Manager/TeamManager.php on line 565 and defined', '/home/user/www/claroline6/claro6-path-lock-dev/Claroline/vendor/claroline/distribution/main/core/Manager/ResourceManager.php', '1813', array()) 
in vendor/claroline/distribution/main/core/Manager/ResourceManager.php at line 181
```

is that because i'm not supposed to create teams in my personnal WS ?

Update : i've manage to create them : i retried, but this time i've given a description...
Weird bug

This however leads to the subsequent bug : https://github.com/claroline/Distribution/issues/581

### PR DESCRIPTION
**Pull request by ngodfraind**
Fixes #630 

The resource query builder could only be used one time per request without buggy behaviour otherwise as it was initialized in the constructor (wich make sense but, the repositories initialized them once in their constructor aswell so it didn't really work as intended).

### CODE REVIEW

**Comment by LaurentGruber** [comment]
LGTM

File: main/core/Manager/ResourceManager.php [hunk 1]
```diff
@@ -1277,7 +1277,9 @@ public function getRoots(User $user)
      */
     public function getWorkspaceRoot(Workspace $workspace)
     {
-        return $this->resourceNodeRepo->findWorkspaceRoot($workspace);
+        $root = $this->resourceNodeRepo->findWorkspaceRoot($workspace);
+
+        return $root;
```
Comment (stefk) [inline main/core/Manager/ResourceManager.php]: No need to introduce a new variable here, original code was ok.

Comment (ngodfraind) [inline main/core/Manager/ResourceManager.php]: I forgot to remove it. It was for debugging.

**Comment by ngodfraind** [comment]
LGTM

**Pull request merged**